client/src/client/user_interface/__init__.py

```diff
@@ -53,16 +61,16 @@ def __init__(
         self.loop.create_task(self._handle_user_input())
         self.cmds_layout: Mapping[Any, Callable[..., Optional[str]]] = {
             # arrow keys
-            self.term.KEY_LEFT:  self.view.layout.cmd_left,
+            self.term.KEY_LEFT: self.view.layout.cmd_left,
             self.term.KEY_RIGHT: self.view.layout.cmd_right,
-            self.term.KEY_DOWN:  self.view.layout.cmd_down,
-            self.term.KEY_UP:    self.view.layout.cmd_up,
+            self.term.KEY_DOWN: self.view.layout.cmd_down,
+            self.term.KEY_UP: self.view.layout.cmd_up,
 
             # arrow keys with shift
-            self.term.KEY_SLEFT:     self.view.layout.cmd_swap_left,
-            self.term.KEY_SRIGHT:    self.view.layout.cmd_swap_right,
-            self.term.KEY_SDOWN:     self.view.layout.cmd_shuffle_down,
-            self.term.KEY_SUP:       self.view.layout.cmd_shuffle_up,
+            self.term.KEY_SLEFT: self.view.layout.cmd_swap_left,
+            self.term.KEY_SRIGHT: self.view.layout.cmd_swap_right,
+            self.term.KEY_SDOWN: self.view.layout.cmd_shuffle_down,
+            self.term.KEY_SUP: self.view.layout.cmd_shuffle_up,
```

Comment:
Your formatter broke my formatting :(

Reply:
I have a default formatter. What should we do with it?

Reply:
I thought I've added `# fmt: skip` there, it is not super important but you can add it again if it dissapeared :P
It's fine as it is, it just looked better I think

Reply:
I am sorry, I've seen this comment too late. Can you please do it? :)